imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -224,6 +221,8 @@ def populate_full_cycle_data(
     -------
     numpy.ndarray
         Array with full cycle data populated.
+    numpy.ndarray
```

Comment:
I think two separate return items is a pattern to avoid if we can. Can you make a dataset that you are passing around as a single object? Or are these two distinct functions that each return one array?

Reply:
I thought about this but I want to keep them in same function because both of them are populated same way and populating them in checker box pattern is complicated. That's why it's helpful to keep them together. I can consolidate return value into new xr.Dataset like you first suggestion. Does that sound ok?

Reply:
Yes, I think keeping structures that are used together in a common "container" like a Dataset makes sense here. Then you can add more named arrays to that in other routines as needed as well but keep passing around one object.

Reply:
I pushed new changes. Can you check if that's what you thought?